lib/galaxy/config/loader.py

```diff
@@ -0,0 +1,128 @@
+import logging
+import os
+
+from galaxy.exceptions import ConfigurationError
+from galaxy.util import string_as_bool
+
+GALAXY_APP_NAME = 'galaxy'
+GALAXY_CONFIG_SCHEMA_PATH = 'lib/galaxy/webapps/galaxy/config_schema.yml'
```

Comment:
The 2 constants above seems not used in this file, remove?

Reply:
Oh yes! (they were used in loader in the first iteration)